ansible/roles/openshift_udp_limits/tasks/main.yml

```diff
@@ -0,0 +1,172 @@
+    # Finally, drop all other UDP traffic. Log dropped traffic up to once per minute.
+    # Because all dropped traffic is crammed into this single limited log rule,
+    # log entries will only be a sampling of actual dropped traffic.
+    - chain: oo-udp
+      after: "id_outbound_udp_filtering"
+      regex: "id_outbound_udp_rule_1_"
+      block: |-
+        -A oo-udp -d 172.16.0.0/12 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -d 204.13.250.23/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -d 204.13.251.23/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -d 208.78.70.23/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -d 208.78.71.23/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -s 10.0.0.0/8 -o eth0 -p udp -j oo-udp-limit
+        -A oo-udp -o eth0 -p udp -m udp --dport 123 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 67 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 161 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 500 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 1514 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 4500 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 25826 -j RETURN
+        -A oo-udp -o eth0 -p udp -m limit --limit 1/min --limit-burst 1 -j LOG --log-prefix "UDP:DROPPED_ALL:"
+        -A oo-udp -p udp -j DROP
+
+    outbound_udp_limit_rules:
+    # Generate per-user outbound UDP limits for every IP in the host subnet
+    - chain: oo-udp-limit
+      after: "id_outbound_udp_rule_1_"
+      regex: "id_outbound_udp_rule_2_"
+      block: |-
+         {%- for i in range(1, (node_cidr | ipaddr('size'))-1) -%}
+         -A oo-udp-limit -s {{ node_cidr | ipaddr(i) | ipaddr('address') }}/32 -m limit --limit 30/sec -j RETURN
+         {% endfor -%}
+
+    # Drop all UDP traffic that exceeds the per-user limits. 
+    # Log dropped traffic up to once per minute.
+    # Because all dropped traffic is crammed into this single limited log rule,
+    # log entries will only be a sampling of actual dropped traffic.
+    - chain: oo-udp-limit
+      after: "id_outbound_udp_rule_2_"
+      regex: "id_outbound_udp_rule_3_"
+      block: |-
+        -A oo-udp-limit -m limit --limit 1/min --limit-burst 1 -j LOG --log-prefix "UDP:DROPPED_FLOOD:"
+        -A oo-udp-limit -j DROP
+
+- name: Create the udp limit chain
+  iptables_chain:
+    name: oo-udp-limit
+    rules: "{{ (outbound_udp_limit_rules|map(attribute='block')|join('')).splitlines() }}"
+
+- name: Create the udp chain
+  iptables_chain:
+    name: oo-udp
+    rules: "{{ (outbound_udp_rules|map(attribute='block')|join('')).splitlines() }}"
+
+- name: List iptables rules filtering chain
+  command: '/usr/sbin/iptables -w -nL OPENSHIFT-OUTPUT-FILTERING'
+  changed_when: False
+  register: filtering_chain_check
+
+- name: Modify in-memory filtering chain
+  command: "/usr/sbin/iptables {{ item.line }}"
```

Comment:
should probably be `"/usr/sbin/iptables -w {{ item.line }}"` so that it'll wait for the lock if something else has it.